HHS/Head-Start-TTADP
# [TTAHUB-2141] revise the "reason" of "Planning/Coordination (also TTA Plan Agreement)" again

## Description of change

Change "Planning/Coordination (also TTA Plan Agreement)" to "Planning/Coordination" in ActivityReports.reason & EventReportPilots.data.reasons

## How to test

run migration, check data in DB for ActivityReports.reason & EventReportPilots.data.reasons
UI will no longer have old value selectable

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2141


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
